core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to process HAR data and timing data"""
+
+import datetime
+import dateutil.relativedelta
+
+
+class ProcessData(object):
+    """Class that contains methods to process data and get us metrics"""
+
+    def __init__(self, har=None, timings=None):
+        self.har = har
+        self.timings = timings
+        self.page_load_timings = None
+        self.result = {
+            'no_requests': -1,
+            'total_size': -1, #in bytes
+            'timings': {
+                'load_time': -1,
+                'dom_ready_time': -1,
+                'request_time': -1
+            }
+        }
+
+    def get_stats(self):
+        """Call all the methods"""
+        self.result['no_requests'] = self._get_no_requests()
+        self.result['total_size'] = self._get_total_size()
+        self.result['timings'] = self._get_timings(self.result['timings'])
+
+    def _get_no_requests(self):
+        no_requests = 0
+
+        if 'log' in self.har:
+            if 'entries' in self.har['log']:
+                no_requests = len(self.har['log']['entries'])
```

Comment:
If this is counting # of requests, why isn't the ``har`` dict reflecting this? Ie, why are they called ``entries`` rather than ``requests``?